src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -2335,8 +1525,6 @@ def _init_weights(self, module: nn.Module):
                 module.weight.data[module.padding_idx].zero_()
 
     def _set_gradient_checkpointing(self, module, value=False):
-        if isinstance(module, MaskFormerSwinEncoder):
-            module.gradient_checkpointing = value
```

Comment:
We still need to do something to enable the gradient checkpointing on the backbone. Like if it's a `MaskformerModel`, call `_set_gradient_checkpointing` on the backbone attribute.

Reply:
Does this look correct to you:
```
 if isinstance(module, MaskFormerPixelLevelModule):
            module.encoder.gradient_checkpointing = value
```
since the backbone is the "encoder" attribute of the `MaskFormerPixelLevelModule` class.